packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -750,9 +751,7 @@ const TRANSFORMS = {
     if (obj.other_config.applies_to) {
       const object = obj.$xapi.getObject(obj.other_config.applies_to, undefined)
       if (object === undefined) {
-        debug(
-          `Unknown other_config.applies_to reference ${obj.other_config.applies_to} in task ${obj.$id}`
-        )
+        debug(`Unknown other_config.applies_to reference ${obj.other_config.applies_to} in task ${obj.$id}`)
```

Comment:
Unrelated changes

Reply:
Prettier indented like this, should i leave or revert it?